engine/conftest.py

```diff
@@ -527,11 +527,11 @@ def _make_slack_user_identity(**kwargs):
 
 @pytest.fixture
 def make_slack_message():
-    def _make_slack_message(channel, alert_group=None, organization=None, **kwargs):
+    def _make_slack_message(slack_team_identity, channel, alert_group=None, **kwargs):
         return SlackMessageFactory(
-            alert_group=alert_group,
-            organization=organization or alert_group.channel.organization,
+            _slack_team_identity=slack_team_identity,
```

Comment:
`channel` already has a reference to `SlackTeamIdentity`, maybe just use it here instead of having an argument for `slack_team_identity`?

Reply:
ahh great point, will implement 👍